lib/src/ffmpeg/filters/custom_filter.dart

```diff
@@ -3,9 +3,10 @@ import 'package:ffmpeg_cli/src/ffmpeg/ffmpeg_command.dart';
 /// Uses the `cliValue` as a literal representation of an FFMPEG
 /// CLI filter, in the case that such a filter is not currently
 /// available as a `Filter` in this package.
-class RawFilter implements Filter {
-  RawFilter(this.cliValue);
+class CustomFilter implements Filter {
```

Comment:
Why was this name changed?

Reply:
Mainly to avoid confusion as there is not a raw filter in ffmpeg, and instead custom filter explains better the function of it. <br>
Also wanted to avoid confusion from the raw file format which this filter has nothing to do with.